packages/devtools_app/test/utils_test.dart

```diff
@@ -417,25 +417,29 @@ void main() {
 
     group('SafeAccess', () {
       test('safeFirst', () {
-        final list = [];
-        expect(list.safeFirst(), isNull);
+        final list = <int>[];
+        final Iterable<int> iterable = list;
```

Comment:
nit: Do we need the Iterable<int> on lvalue with list = <int>[] inference?

Reply:
the cast to I do want to test on Iterable to make sure the extension methods are applying to Iterable as well as List. Maybe that doesn't need to be tested but i think it is a reasonable sort of thing to verify given that extension method behavior does vary depending on the type of the target.